tsig_test.go

```diff
@@ -231,9 +232,128 @@ func TestTSIGHMAC224And384(t *testing.T) {
 			if mac != tc.expectedMAC {
 				t.Fatalf("MAC doesn't match: expected '%s' but got '%s'", tc.expectedMAC, mac)
 			}
-			if err = tsigVerify(msgData, tc.secret, "", false, timeSigned); err != nil {
+			if err = tsigVerify(msgData, "", false, timeSigned, tsigHMACProvider(tc.secret)); err != nil {
 				t.Error(err)
 			}
 		})
 	}
 }
+
+const (
+	testGoodKeyName = "goodkey."
```

Comment:
Just use the `const testGoodKeyName = "goodkey."` form here.

Reply:
I think I might have had multiple constants at one point and `go fmt` doesn't collapse back to the single form. Sorted.